src/api/authApi.js

```diff
@@ -0,0 +1,29 @@
+import springApi from './springApi';
+
+export const authAPI = {
+  login({ userName, password }) {
+    return springApi
+      .post('/login', { userName, password })
+      .then(res => res)
+      .catch(error => error.response);
+  },
+  signUp({ userName, password, repeatPassword, firstName, lastName, age }) {
+    return springApi
+      .post('/registration', {
+        userName,
+        password,
+        repeatPassword,
+        firstName,
+        lastName,
+        age
+      })
+      .then(res => res)
+      .catch(error => error.response);
+  },
+  logout() {
+    return springApi.delete('/refresh_token');
```

Comment:
REST API paths should not have underscore. Use hyphen instead. Might be worth to read this https://restfulapi.net/